doc/classes/Object.xml

```diff
@@ -192,6 +192,55 @@
 				[b]Note:[/b] If [method _init] is defined with [i]required[/i] parameters, the Object with script may only be created directly. If any other means (such as [method PackedScene.instantiate] or [method Node.duplicate]) are used, the script's initialization will fail.
 			</description>
 		</method>
+		<method name="_iter_get" qualifiers="virtual">
+			<return type="Variant" />
+			<param index="0" name="iter" type="Variant" />
+			<description>
+				Returns the current iterable value. [param iter] stores the iteration state, but unlike [method _iter_init] and [method _iter_next] the state is supposed to be read-only, so there is no [Array] wrapper.
+			</description>
+		</method>
+		<method name="_iter_init" qualifiers="virtual">
+			<return type="bool" />
+			<param index="0" name="iter" type="Array" />
+			<description>
+				Initializes the iterator. [param iter] stores the iteration state. Since GDScript does not support passing arguments by reference, a single-element array is used instead. Returns [code]false[/code] if the current iteration is the last one.
```

Comment:
```suggestion
				Initializes the iterator. [param iter] stores the iteration state. Since GDScript does not support passing arguments by reference, a single-element array is used as a wrapper. Returns [code]true[/code] if the iterator has not yet reached the end.
```

Reply:
Thank you @dalexeev for suggesting the changes to @dalexeev I hope @dalexeev implements them when necessary

Reply:
I guess @dalexeev just wasn't sure about these changes and left comments so he wouldn't forget to fix them later. Do you think these are good changes? If so, I'll tell @dalexeev and he'll apply them.

Reply:
Thanks @dalexeev , tell them that I think saying "as a wrapper" is probably a good idea, as using arrays like this is pretty uniquely unusual and is worth specifying. As for the `true` VS. `false` change, I am not sure. They both have their merits. Maybe...?
```suggestion
				Initializes the iterator. [param iter] stores the iteration state. Since GDScript does not support passing arguments by reference, a single-element array is used as a wrapper. Returns [code]true[/code] so long as the iterator has not reached the end.
```